db/migrate/20171108010437_add_married_to_member.rb

```diff
@@ -0,0 +1,10 @@
+class AddMarriedToMember < ActiveRecord::Migration[5.1]
+  def up
+    add_column :members, :married, :boolean
```

Comment:
random thought I had (in my sleep!) about this: why not use the `marital_status` column that we already have on `members` and just updated to `Married` ?